src/app/user/user-form.component.html

```diff
@@ -146,12 +146,6 @@
 </div>
 
 <div class="flex row" *ngIf="admin">
-  <md-checkbox [(ngModel)]="model.notifications.schedule" name="notify-schedule">
```

Comment:
I'll open a PR for the refactor of this afterwards, I just wanted to split them up two not have one branch with lots of changes